src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -0,0 +1,38 @@
+package zio.redis.api
```

Comment:
I might have a package object in `zio.redis.api` with:

```scala
package object api {
  val sortedsets = new SortedSets {}
  val sets = new Sets {}
}
```

For more "ala carte" imports.